# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../arping/usrlocal.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- ./Makefile.orig	2004-05-23 00:51:39.000000000 +0200
+++ ./Makefile	2004-05-23 00:52:08.000000000 +0200
@@ -83,8 +83,8 @@
 	$(MAKE) USE_NETIF=0 SOLARIS=1 arping1-make
 
 install:
-	install -c arping /usr/local/bin/arping
-	install arping.8 /usr/local/man/man8/arping.8
+	install -c arping /usr/bin/arping
+	install arping.8 /usr/man/man8/arping.8
 
 arping.o: arping.c
 	$(CC) -Wall $(CFLAGS) -c `libnet-config --defines` `libnet-config --cflags` arping.c
